opentelemetry/src/global/trace.rs

```diff
@@ -370,6 +363,10 @@ impl GlobalTracerProvider {
             provider: Arc::new(provider),
         }
     }
+
+    fn force_flush(&self) -> Vec<TraceResult<()>> {
+        Vec::new()
```

Comment:
Yeah, I guess we need to first figure out whether `global` is part of SDK or API. i.e should we allow users to provide their own implementation of TracerProvider

Reply:
I think we need to allow users to bring their own implementation and be able to use `global`. In that case, we need to downcast the `Box<dyn TracerProvider>`. That may cause some perf issue as it's dynamic reflection. @jtescher any thoughts?

Reply:
@TommyCpp - i believe we can ask the users to use `TracerProvider::Create` rather than providing option for users to create custom tracer providers because the issue linked was to decouple `TracerProvider::Create` from the underlying functionality.
Exposing ability to create custom TracerProviders also will include exposing traits that provide SDK functionality, 
which is opposite of what we have done as part of this issue.

Reply:
I believe the `global` is part of the API. As the API asks implementation to provide a way to register any `TracerProvider` in API. 
> Normally, the TracerProvider is expected to be accessed from a central place. Thus, the API SHOULD provide a way to set/register and access a global default TracerProvider

To that end, we should probably remove `force_flush` method from the `global` module. However, I believe many users want to use `force_flush` to make sure their spans are sent.

Reply:
Got it. Thanks

Reply:
Yeah let's remove it for now as it's not in the API spec